code-2.java

```diff
@@ -1,153 +0,0 @@
-                System.out.println("You guessed incorrectly. Minus one guess");
-                guessesRemaining--;
-            }
-
-            // Print state of game board
-            printGameBoard(gameBoard);
-            System.out.println("Lives remaining: " + String.valueOf(guessesRemaining));
-
-            // Check win and loss condition
-            if(guessesRemaining <= 0){
-                System.out.println("GAME OVER. Out of lives");
-                game_over = true;
-                break;
-            }
-
-            if(String.valueOf(gameBoard).equals(String.valueOf(GAME_WORD))){
-                System.out.println("CONGRATULATIONS YOU WON");
-                game_over = true;
-                break;
-            }
-        }
-    }
-
-    /**
-     * Helper function to print the game board with spaces between letters
-     * @param gameBoard
-     */
-    private static void printGameBoard(char[] gameBoard){
-        for(char c : gameBoard)
-            System.out.print(c + " ");
-        System.out.println();
-    }
-
-
-    /**
-     * Helper function for processing a guessed letter
-     * @param letter - letter that user provided as guess
-     * @param gameBoard - gameBoard used to represent state of guessed characters and unknown
-     * @param GAME_WORD - the selected GAME_WORD the user is trying to guess
-     * @return boolean - return true if user guessed a letter correctly
-     */
-    private static boolean guessLetter(String letter, char[] gameBoard, char[] GAME_WORD){
-        boolean correctGuess = false;
-
-        // Iterate through gamestate and reveal all correctly guessed letters
-        for(int itr = 0; itr < GAME_WORD.length; itr++){
-            char guessedChar = letter.charAt(0);
-            // Check this letter with the string
-            if(GAME_WORD[itr] == guessedChar){
-                gameBoard[itr] = guessedChar;
-                correctGuess = true;
-            }
-        }
-
-        return correctGuess;
-    }
-
-
-    /**
-     * Helper function to validate input from user
-     * @param input - String read in by Scanner.next
-     * @return boolean - true for valid input
-     */
-    private static boolean validInput(String input){
-        // Provided no input
-        if(input.length() == 0 || input == null){
-            return false;
-        }
-
-        // Provided more than one letter
-        if(input.length() > 1){
-            return false;
-        }
-
-        // Input is not a letter
-        if(!Character.isLetter(input.charAt(0))){
-            return false;
-        }
-
-        return true;
-    }
-
-
-    /**
-     * Helper function to randomly select a word
-     * @return char[] - returns a randomly selected word as char array
-     */
-    private static char[] generateGameWord(){
-        // Create list to store game words, using arraylist allows for dynamically adding words instead of static array
-        List<String> WORDS = new ArrayList<String>();

```

Comment:
Are you going to be dynamically adding words though at run-time? If not, I'd rather keep this as a constant.